fleetdm/fleet
# Adding extension_id and browser columns to software table.

https://github.com/fleetdm/confidential/issues/2118
API doc changes are at: https://github.com/fleetdm/fleet/pull/15092

This change affects Chrome-based and Firefox browsers. I tried Safari, but it appears that latest Safari extensions simply show up as regular apps on macOS.

# Checklist for submitter
  - Tested detail query changes on macOS, Windows, Ubuntu, and ChromeOS
  - Tested Chrome, Edge, Brave, and Firefox browsers

Testing notes:
1. Install some browser extensions.
2. Update fleet to this change.
3. Refresh hosts, and trigger vulnerabilities job (which updates host software).
4. Do something like `GET https://localhost:8080/api/v1/fleet/software?query=Password%20Manager` to see `extension_id` and `browser` for the hosts that have updated.
